src/jobs/jobs.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@nestjs/common'
+import axios from 'axios'
+import { config } from 'dotenv'
+
+config()
```

Comment:
this is probably why we need a configurationModule. We need to do this every time we try to access an env variable

Reply:
I saw this.. 

I could've sworn as long as `config()` is run once in the `main.ts` file, the `process.env` values _should_ be available everywhere.. 🤔

Reply:
oooooh.. NestJS is loading the service files _before_ it's loading `main.ts`!

```
jobs.service.ts
process.env.LOG_LEVEL => debug
main.ts
process.env.LOG_LEVEL => debug
```

There's gotta be a way to do this somewhere that will run first, before everything else.